src/main.h

```diff
@@ -211,7 +215,7 @@ std::string GetWarnings(std::string strFor);
 /** Retrieve a transaction (from memory pool, or from disk, if possible) */
 bool GetTransaction(const uint256 &hash, CTransaction &tx, uint256 &hashBlock, bool fAllowSlow = false);
 /** Find the best known block, and make it the tip of the block chain */
-bool ActivateBestChain(CValidationState &state, CBlock *pblock = NULL);
+bool ActivateBestChain(CValidationState &state, CBlock *pblock = NULL, libzcash::ProofVerifier& verifier = disabledVerifier);
```

Comment:
Move the `verifier` parameter one slot to the left, so it doesn't need a default value. We could even do this in every function (have `ProofVerifier` right next to `CValidationState`) for consistency.